app/assets/javascripts/modules/coronavirus-local-restrictions.js

```diff
@@ -0,0 +1,37 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function CoronavirusLocalRestrictions () {}
+
+  CoronavirusLocalRestrictions.prototype.start = function (element) {
+    this.addPostcodeFormSubmitListener(element[0])
+    this.addErrorMessageEvent(element[0])
+  }
+
+  CoronavirusLocalRestrictions.prototype.addPostcodeFormSubmitListener = function (element) {
+    element.querySelector('.coronavirus-local-restrictions__postcode-form')
```

Comment:
Ideally, we would've attached the module script right on the form element without needing to query this element, but I do get that we're going it at a higher level so we can grab the error message too.

Reply:
I did this initially then realised that had to add in multiple events so changed it to page level